conjur_iam_client.py

```diff
@@ -78,7 +78,7 @@ def create_conjur_iam_api_key(iam_role_name=None, access_key=None, secret_key=No
         secret_key = os.environ["AWS_SECRET_ACCESS_KEY"]
         token = os.environ["AWS_SESSION_TOKEN"]
 
-    region = os.environ["AWS_REGION"]

```

Comment:
Did you test this in a CyberArk-provided AWS account? I have a feeling that this is a security constraint of a CyberArk-provided AWS account and not generally the case that only us-east-1 works with AWS STS in Lambda. If you were using one, I can test in other regions myself.

Reply:
I did, but STS endpoint is enabled in the region I deployed into. I’ll do some digging.

Reply:
I'll do the same myself and see what I can come up with...